.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -18,31 +18,43 @@ body:
 - type: input
   attributes:
     label: OpenRCT2 build
-    placeholder: e.g. OpenRCT2, v0.3.4.1 (d76611e on develop) provided by GitHub
+    placeholder: e.g. OpenRCT2, v0.4.2-53-g3fa7d59 (3fa7d59 on develop) provided by GitHub
     description: You can see the version by running `openrct2 --version` or copy it from the About window. Open the "Disk and game options" menu (floppy disk on the top toolbar), choose "About 'OpenRCT2'" and click the copy button next to the version information.
   validations:
     required: true
 
-- type: textarea
+- type: dropdown
   attributes:
-    label: Describe the issue
-    description: Please describe the issue you are experiencing here.
+    label: Basegame
```

Comment:
I don't think `basegame` is a real word. Below, you use the term `original game`. I think that one fits better here too.

Reply:
Original game sounds more like if it occurs in what game even though neither is the game you are playing. Base game is what game you have chosen as game to load OpenRCT2 up with. I'll gladly split the words of, I think it's the Dutch in me combine words :P

Reply:
Alright, I agree. Just split the words :)